### DISCUSSION TITLE
RNMBXMapViewModule error help

### DISCUSSION THREAD

**Discussion by HKevinH** [rnmapbox/maps#3264] [Q&A]
# Environment
## Dev : Windows 11

TurboModuleRegistry.getEnforcing(...): 'RNMBXMapViewModule' could not be found. Verify that a module by this name is registered in the native binary., js engine: hermes

https://github.com/rnmapbox/maps/issues/3193

  "dependencies": {
    "@expo/webpack-config": "^19.0.0",
    "@react-native-async-storage/async-storage": "1.18.2",
    "@react-navigation/elements": "^1.3.21",
    "@react-navigation/native": "^6.1.9",
    "@react-navigation/stack": "^6.3.20",
    "@rnmapbox/maps": "^10.1.0-beta.23",
    "axios": "^1.6.2",
    "babel-preset-expo": "^9.5.2",
    "deprecated-react-native-prop-types": "^5.0.0",
    "expo": "^49.0.21",
    "expo-app-loading": "^2.1.1",
    "expo-location": "~16.1.0",
    "expo-splash-screen": "~0.20.5",
    "expo-status-bar": "~1.6.0",
    "firebase": "^10.7.1",
    "react": "18.2.0",
    "react-native": "0.72.6",
    "react-native-gesture-handler": "~2.12.0",
    "react-native-maps": "1.7.1",
    "react-native-safe-area-context": "4.6.3",
    "react-native-screens": "~3.22.0",
    "react-native-vector-icons": "^10.0.2",
    "react-native-web": "~0.19.6",
    "react-native-webview": "13.2.2",
    "styled-components": "^6.1.1",
    "uglify-js": "^3.17.4",
    "uuid": "^9.0.1",
    "xmldom": "^0.6.0",
    "zustand": "^4.4.7"
  },

**Comment by mfazekas** [comment]
What is your expo config, have you built your expo app, how? We don't support expo go.

**Comment by HKevinH** [comment]
npx create-expo-app
npm install --save @rnmapbox/maps

**Comment by HKevinH** [comment]
![image](https://github.com/rnmapbox/maps/assets/65212876/5072a107-a4e7-4169-8b52-3d5f47f5dac4)
